mod-linked-data/src/main/resources/citation/mod-linked-data/features/util/crud-resource.feature

```diff
@@ -44,4 +44,41 @@ Feature: CRUD operations on a resource
     Given path 'inventory/instances/' + inventoryInstanceId
     And request inventoryInstance
     When method put
-    Then status 204
\ No newline at end of file
+    Then status 204
+
+  @postAuthorityFile
+  Scenario: Post an authority file
+    Given path 'authority-source-files'
+    And request authorityFileRequest
+    When method POST
+    Then status 201
+
+  @postAuthority
```

Comment:
maybe instead of creating this one we can reuse ```@postBibToSrs``` (can be renamed accordingly)

Reply:
that's right, fixed, thanks!